src/styles/base/_base.scss

```diff
@@ -1,19 +1,35 @@
-@use '../abstracts/mixins' as mixins;
-@use '../abstracts/variables' as *;
+@use '../abstracts' as *;
 
 .container {
-    background-color: hsl(60, 100%, 70%);
-    height: 100vh;
     margin-inline: auto;
     max-width: map-get($map: $screen-max-width, $key: 'small');
 
-    @include mixins.media-up(map-get($map: $screen-size, $key: 'medium')) {
-        background-color: hsl(240, 100%, 70%);
+    @include media-up(map-get($map: $screen-size, $key: 'medium')) {
         max-width: map-get($map: $screen-max-width, $key: 'medium');
     }
 
-    @include mixins.media-up(map-get($map: $screen-size, $key: 'large')) {
-        background-color: hsl(120, 100%, 70%);
+    @include media-up(map-get($map: $screen-size, $key: 'large')) {
         max-width: map-get($map: $screen-max-width, $key: 'large');
     }
 }
+
+.icon {
+    font-size: 2.5rem;
+}
+
+body {
+    height: 300vh;
```

Comment:
why is this needed? If needed for testing, just use it for local and remove it when pushing code to git